scripts/cat/cats.py

```diff
@@ -412,8 +412,14 @@ def die(self, body: bool = True):
 
         May return some additional text to add to the death event.
         """
-        self.injuries.clear()
-        self.illnesses.clear()
+        if self.status == 'leader' and 'pregnant' in self.injuries and game.clan.leader_lives > 0:
```

Comment:
Shouldn't this be `game.clan.leader_lives >= 0` ? To match the check below. As it is, this will only trigger if the game.clan.leader_lives is -1 or less.